docs/src/main/sphinx/security/user-mapping.rst

```diff
@@ -95,9 +95,9 @@ also maps users in the form ``bob@uk.example.com`` to ``bob_uk``:
 .. literalinclude:: user-mapping.json
     :language: json
 
-Configure this example file to use the :doc:`LDAP </security/ldap>`
-authentication method (which uses the PASSWORD authentication type), as
-follows:
+Set up the preceding example to use the :doc:`LDAP </security/ldap>`
+authentication method with the PASSWORD authentication type by adding
```

Comment:
nit: link PASSWORD to https://trino.io/docs/current/security/authentication-types.html#security-authentication-types--page-root maybe?

Reply:
Done.